src/mpl2/src/pin_alignment.cpp

```diff
@@ -380,6 +380,9 @@ void SimulatedAnnealingCore::Restore()
 void SimulatedAnnealingCore::CalculateOutlinePenalty()
 {
   outline_penalty_ = 0.0;
+  
+  float ar = height_ / width_ - outline_height_ / outline_width_;
+  //outline_penalty_ = ar * ar;
```

Comment:
This looks to be dead code.  I don't see any use of ar

Reply:
removed ar for calculation of outline_penalty